addons/event/models/event_mail.py

```diff
@@ -118,11 +160,18 @@ def execute(self):
                     mail.write({'mail_registration_ids': lines})
                 # execute scheduler on registrations
                 mail.mail_registration_ids.filtered(lambda reg: reg.scheduled_date and reg.scheduled_date <= now).execute()
+
+                contacted_attendees = len(mail.mail_registration_ids.filtered(lambda reg: reg.mail_sent))
+                mail.sent_to_message = _('%i Attendees') % contacted_attendees if contacted_attendees else False
```

Comment:
It is a bit strange that this will be translated into language of cron user probably (aka, admin). We could try to either store only the count (and add strings in views, but notably in embedded list views it is sometimes complicated to have something nice), either translate it into responsible's language.

Same for below.

Reply:
Translated into the language of the responsible :)